dbt_integration.py

```diff
@@ -169,17 +169,19 @@ def wrapper(
     return wrapper
 
 
-def default_profiles_dir(project_dir: Path) -> Path:
+def default_profiles_dir(project_dir) -> str:
     if "DBT_PROFILES_DIR" in os.environ:
-        return Path(os.environ["DBT_PROFILES_DIR"]).resolve()
+        return str(Path(os.environ["DBT_PROFILES_DIR"]).resolve())
     return (
-        project_dir if (project_dir / "profiles.yml").exists() else Path.home() / ".dbt"
+        str(project_dir)
+        if (project_dir / "profiles.yml").exists()
+        else str(Path.home() / ".dbt")
```

Comment:
That won't work for windows

Reply:
yes, I was thinking of handling it in the python level itself, or just avoiding `str` altogether